curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/pleasure-to-meet-you.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 65528a7c39eb38f51e0e6f7f
+title: Pleasure to Meet You
+challengeType: 19
+dashedName: pleasure-to-meet-you
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+Expressions like `It's a pleasure to meet you` are commonly used when meeting someone for the first time to show politeness and appreciation when meeting someone for the first time.
```

Comment:
```suggestion
Expressions like `It's a pleasure to meet you` are commonly used when meeting someone for the first time as a polite way to greet someone.
```
The earlier part of the sentence already has "meeting someone for the first time" and repeating it feels a bit clunky.